.ls-lint.yml

```diff
@@ -0,0 +1,19 @@
+ls:
+  .dir: regex:([.a-z0-9]*)([-.][a-z0-9]+)*
+  .css: regex:([.a-z0-9]*)([-.][a-z0-9]+)*
+  .ts: regex:([.a-z0-9]*)([-.][a-z0-9]+)*
+  .tsx: regex:([.a-z0-9]*)([-.][a-z0-9]+)*
```

Comment:
We can use `kebab-case`, instead of regex
https://ls-lint.org/1.x/configuration/the-rules.html

Reply:
I do not think it is an option. These RegExp also covers case when the file is named like a-b.c.ts but kebab-case will not